modules/ROOT/pages/mergetags.adoc

```diff
@@ -101,6 +101,19 @@ Here is an example of the {pluginname} HTML structure.
 </span>
 ----
 
+== Merge Tags and the Insertion Point Marker
```

Comment:
Actually, the Merge Tags plugin knows nothing about {{mce-cursor}} marker, and the editor content may contain such a merge tag. However, the Advanced Template plugin just replaces the marker with a special hook element before inserting the template in the editor content.

Reply:
To quote my commit message:

> Notes on the relationship (or lack thereof) between the Insertion Point Marker string and the Merge Tags plugin re-written.